### PR TITLE
[Documentation][Customization] Overriding entire grids

### PR DESCRIPTION
**Pull request by TheMadeleine**
| Q               | A
| --------------- | ---
| Doc fix?        | no |
| New docs?    | yes |
| Related tickets |  |
| License         | MIT |

### CODE REVIEW

File: docs/customization/grid.rst [hunk 1]
```diff
@@ -1,26 +1,28 @@
...
-* override entire grid
+* override an entire grid
 
 How to customize grids?
 -----------------------
 
 .. tip::
 
-    First of all if you are attempting to change anything in any state machine in **Sylius** you will need a special file:
+    One way to change anything in any state machine in **Sylius** is to create a special file in the ``app/config/`` directory:
```
Comment (tuka217) [inline docs/customization/grid.rst]: Should it be `in any state machine`?

Comment (TheMadeleine) [inline docs/customization/grid.rst]: ?

Comment (TheMadeleine) [inline docs/customization/grid.rst]: Damn. :D `in any grid` of course xD

**Comment by pjedrzejewski** [comment]
@TheMadeleine Requires rebasing. :)

**Comment by TheMadeleine** [comment]
Rebased already @pjedrzejewski ;)

**Comment by pjedrzejewski** [comment]
Thanks Magda!

**Pull request merged**